tchannel/tornado/dispatch.py

```diff
@@ -71,13 +73,12 @@ def __init__(self, _handler_returns_response=False):
 
     _HANDLER_NAMES = {
         Types.CALL_REQ: 'pre_call',
-        Types.CALL_REQ_CONTINUE: 'pre_call'
+        Types.CALL_REQ_CONTINUE: 'pre_call_cont'
     }
```

Comment:
I don't really like this `getattr` based handling scheme we have going on there. Let's just map message types to methods directly. At the bottom of the class, after all the defs, do:

    _HANDLERS = {
        Types.CALL_REQ: handle_pre_call,
        ...
    }

And in `handle`, do

    self._HANDLERS[message.message_type](self, message, connection)